integrasjon/rest-klient/src/main/java/no/nav/vedtak/felles/integrasjon/rest/RestClientSupportProdusent.java

```diff
@@ -32,12 +32,10 @@
 
     private final OidcRestClient oidcRestClient;
     private final SystemUserOidcRestClient systemUserOidcRestClient;
-    private final UserAndSystemOidcRestClient userAndSystemOidcRestClient;
 
     public RestClientSupportProdusent() {
         this.oidcRestClient = createOidcRestClient();
         this.systemUserOidcRestClient = creatSystemUserOidcRestClient();
```

Comment:
Igjenm bedre med en generell klient og forskjellige interceptorer ?

Reply:
Trenger å reflektere over hele opplegget rundt RS-integrasjon. Ditt for WS og felles-sikkerhet
Patternet hindrer utvidelse og tilpasning og oppfordrer til ymse veier rundt.